src/command/mcl.zig

```diff
@@ -10,6 +10,35 @@ var line_accelerations: []u7 = undefined;
 
 const Direction = mcl.Direction;
 const Station = mcl.Station;
+// const Registers = struct {
+//     x: mcl.registers.X,
+//     y: mcl.registers.Y,
+//     wr: mcl.registers.Wr,
+//     ww: mcl.registers.Ww,
+// };
+
+// const RegisterType = enum { x, y, wr, ww };
+// var register_log_file: ?std.fs.File = null;
+// var register_list = std.EnumArray(RegisterType, bool).initFill(false);
+// var station_id_log: std.ArrayList(u8) = undefined;
+
+const RegisterLogging = struct {
+    line_name: []const u8,
+    register_list: std.EnumArray(RegisterType, bool), // Specify which register to log for each line
+    station_id: []u8, // Specify the station to log based on axis provided by user
```

Comment:
Slice should have a plural field name. 
Additionally, while keeping the slice is good for convenient and type-safe access, we do not need to make it dynamically allocated; we're guaranteed that one line can contain a maximum of 256 stations.

Finally, make this a slice of indices instead of IDs; the maximum supportable ID value is 256, which does not fit into a `u8`.